internal/server/data/settings.go

```diff
@@ -65,8 +65,9 @@ func GetSettings(db GormTxn) (*models.Settings, error) {
 
 func getSettingsForOrg(tx GormTxn, orgID uid.ID) (*models.Settings, error) {
 	db := tx.GormDB()
+	db = ByOrgID(orgID)(db)
```

Comment:
Maybe not related to this change, but looks like we should use the standard `get` function here rather than the `db.First`?

Reply:
Ya, seems like that might work. I'm not sure why it's different.  Either way we'll be moving away from `get` very soon when we replace gorm queries with sql. In the new sql version I don't think we'll have a `get` function because there's nothing really common to selecting from different tables other than error handling.